roles/pulp_health_check/tasks/main.yml

```diff
@@ -0,0 +1,9 @@
+---
+- name: Get Pulp services
+  service_facts:
+
+- name: Checking Pulp services
+  fail:
+    msg: "{{ item.value.name }} state: {{ item.value.state }}"
+  when: "'pulp' in item.value.name and item.value.name != 'pulpcore-worker@.service' and item.value.state != 'running'"
```

Comment:
```
           "pulpcore-api.service": {
                "name": "pulpcore-api.service",
                "source": "systemd",
                "state": "running",
                "status": "enabled"
            },
            "pulpcore-content.service": {
                "name": "pulpcore-content.service",
                "source": "systemd",
                "state": "running",
                "status": "enabled"
            },
            "pulpcore-resource-manager.service": {
                "name": "pulpcore-resource-manager.service",
                "source": "systemd",
                "state": "running",
                "status": "enabled"
            },
            "pulpcore-worker@.service": {
                "name": "pulpcore-worker@.service",
                "source": "systemd",
                "state": "unknown",
                "status": "enabled"
            },
            "pulpcore-worker@1.service": {
                "name": "pulpcore-worker@1.service",
                "source": "systemd",
                "state": "running",
                "status": "unknown"
            },
            "pulpcore-worker@2.service": {
                "name": "pulpcore-worker@2.service",
                "source": "systemd",
                "state": "running",
                "status": "unknown"
            },
```

Reply:
removing `pulpcore-worker@.service` as I'm not sure if the state should be `running`